tests/scripts/thread-cert/Cert_5_3_07_DuplicateAddress.py

```diff
@@ -117,7 +117,13 @@ def test(self):
         self.nodes[ED2].add_ipaddr('2001::1')
         time.sleep(3)
 
-        self.nodes[ED3].ping('2001::1')
+
+        # Final ping doesn't need a response;
+        # used to trigger an ADDR_ERR.ntf
+        try:
+            self.nodes[ED3].ping('2001::1')
+        except pexpect.TIMEOUT:
```

Comment:
In the NCP configuration, this assumption fails: no response is raised to the host and the ping timeout is exceed.  It seems the intent of this test is to verify that ADDR_ERR.ntf is sent and not that a ping response comes in during a duplicate address situation.

We can investigate why NCP and raw CLI behavior is different here, but I don't think it is worth blocking addition of continuous integration of Spinel NCP configurations for a check that isn't actually part of the certification test plan.